end-to-end/test/basic.test.ts

```diff
@@ -59,6 +59,44 @@ describe('basic Full Client tests', () => {
     expect(resources[0].type).toBe('Document');
   });
 
+  it('allows you to block urls', async () => {
+    koaServer.get('/block', ctx => {
+      ctx.body = `<html>
+<head>
+  <link rel="stylesheet" href="/test.css" />
+</head>
+<body>
+  <img src="/img.png" alt="Image"/>
+</body>
+</html>`;
+    });
+
+    koaServer.get('/img.png', ctx => {
+      ctx.statusCode = 500;
+    });
+    koaServer.get('/test.css', ctx => {
+      ctx.statusCode = 500;
+    });
+
+    const hero = new Hero({
+      blockedResourceUrls: [
+        // eslint-disable-next-line no-useless-escape
+        '\*\.png',
```

Comment:
Are you trying to intentionally test that any url with *.png in it will be blocked (not a wildcard, but the actual character?). It would be nice to put one of these urls behind a more complicated pattern with a ? or another troublemaker character and show that it works.

Reply:
this was just the easiest way I saw to do something, as for anything more I probably need t add some resources to the server, no? Or how does it work?

Reply:
```koaServer.get('/img.png', ctx => {
       ctx.statusCode = 500;
     });```

That's the route definition. so just make it something more complicated.